src/org/zaproxy/zap/extension/ascanrules/TestSQLInjection.java

```diff
@@ -1472,6 +1467,46 @@ protected String stripOff(String body, String pattern) {
 	}
 
 	/**
+	 * Replace body by stripping of pattern string. The URLencoded pattern will
+	 * also be stripped off. The URL decoded pattern will not be stripped off,
+	 * as this is not necessary of rour purposes, and causes issues when
+	 * attempting to decode parameter values such as '%' (a single percent
+	 * character) This is mainly used for stripping off a testing string in HTTP
+	 * response for comparison against the original response. Reference:
+	 * TestInjectionSQL
+	 *
+	 * @param body
+	 * @param pattern
+	 * @return
+	 */
+	protected String stripOffOriginalParam(String body, String pattern) {
+		if (pattern == null) {
+			return body;
+		}
+
+		String urlEncodePattern = getURLEncode(pattern);
+		String result = body.replaceAll("\\Q" + pattern + "\\E", "").replaceAll("\\Q" + urlEncodePattern + "\\E", "");
+		return result;
+	}
+
+	/**
+	 * Replace body by stripping of pattern strings.
+	 *
+	 * @param body
+	 * @param original pattern
+	 * @param attack pattern
+	 * @return
```

Comment:
There's no point including these 3 or 4 lines if they're not going to be defined/documented.